polaris-core/src/main/java/org/apache/polaris/core/entity/CatalogEntity.java

```diff
@@ -79,7 +79,7 @@ public static CatalogEntity of(PolarisBaseEntity sourceEntity) {
     return null;
   }
 
-  public static CatalogEntity fromCatalog(CallContext callContext, Catalog catalog) {
+  public static CatalogEntity fromCatalog(RealmConfig realmConfig, Catalog catalog) {
```

Comment:
nit: I tend to think the PR title should mention "replace CallContext with RealmConfig ..." :)